pkg/ossm/smcp_install.go

```diff
@@ -22,10 +22,10 @@ import (
 	"github.com/maistra/maistra-test-tool/pkg/util"
 )
 
-func installDefaultSMCP22() {
```

Comment:
we still need the installDefaultSMCP22 functions in a 2.3 branch testing. 
We can copy that function and then apply your new version changes.

Reply:
oh, wait. that's the default SMCP. Ignore my comment above.

Reply:
Added again